src/main/java/nl/tudelft/pixelperfect/gamestates/WonState.java

```diff
@@ -0,0 +1,40 @@
+package nl.tudelft.pixelperfect.gamestates;
+
+import nl.tudelft.pixelperfect.Game;
+
+/**
+ * Created by woute on 5/23/2016.
+ */
+public class WonState extends GameState {
+
+  /**
+   * Constructor for WonState.
+   * @param game
```

Comment:
Please add a space between the javadoc description and the javadoc input/param/etc list.